### PR TITLE
Move freedesktop.org/wiki/HarfBuzz to README.md

### PR DESCRIPTION
**Pull request by khaledhosny**
I copied the text verbatim, but it probably needs to be edited. I’m not sure if there is more to move here.

### CODE REVIEW

**Comment by khaledhosny** [comment]
I think I mixed things and this page is not what harfbuzz.org used to point to (now I see why it felt so thin in details).

Yup, this one is more like it https://web.archive.org/web/20190406161058/https://www.freedesktop.org/wiki/Software/HarfBuzz/

File: README.md [hunk 1]
```diff
@@ -6,26 +6,94 @@
...
+
+Both development and user support discussion around HarfBuzz happens on the
+[github][4].
+
+To report bugs or submit patches please use [github][4] issues and
+pull-requests.
+
+For a comparison of old vs new HarfBuzz memory consumption see [this][10].
+
+<!--See past and upcoming [HarfBuzz Hackfests](https://freedesktop.org/wiki/Software/HarfBuzz/Hackfests/)!-->
```
Comment (khaledhosny) [inline README.md:66]: @behdad Do we want the contents of this page as well?

**Comment by behdad** [comment]
Thanks. Some of this info is obsolete or not of interest anymore. Like the hackfests, the pango-vs-harfbuzz. Let's shorten it a bit.

File: README.md (hunk 1 above)
Comment (behdad) [reply README.md:66]: Don't think so.

**Comment by behdad** [comment]
Let's merge and cleanup after.

**Pull request merged**